tests/test_integration_1.py

```diff
@@ -43,22 +43,25 @@ def test_loading_slider(self):
 
         app = dash.Dash(__name__)
 
-        app.layout = html.Div([
-            html.Button(id='test-btn'),
-            html.Label(id='test-div', children=['Horizontal Slider']),
-            dcc.Slider(
-                id='horizontal-slider',
-                min=0,
-                max=9,
-                marks={i: 'Label {}'.format(i) if i == 1 else str(i)
-                       for i in range(1, 6)},
-                value=5,
-            ),
-        ])
+        app.layout = html.Div(
```

Comment:
The only manual changes here were removing the `output=`, `input=`, `state=` keyword arguments in the `test_location_link` test in order to work around plotly/dash#1366; the other changes (double `"` quotes, indentation) are formatting changes from `black`ening this file.